### DISCUSSION TITLE
Screening - prior knowledge & stopping rule

### DISCUSSION THREAD

**Discussion by cnslkrks** [asreview/asreview#1982] [Q&A]
I plan to use ASReview for the screening phase of my systematic review paper. I would like to know if there are any guidelines for determining the number of articles that should be manually labeled as relevant or irrelevant for ASreview to use as prior knowledge. I’m also curious about how many consecutive articles should be labeled as irrelevant before the screening process can be considered complete (stopping criteria). I was wondering if there is any rule of thumb to determine these numbers. Any help is appreciated!

**Comment by lastoel** [comment]
Hi there! You have two questions, one about prior knowledge, and one about how to determine the stopping rules for your screening process. 

1. For ASReview to work, you need to label **at least** one relevant and one irrelevant record as prior knowledge. Based on that, the active learning cycle can start. You can label more records as prior knowledge if you want to, but it's not a necessity. 
2. There is no 'golden standard' number of consecutive irrelevant articles that can be defined as the Stopping Rule to Rule them all ;) It heavily depends on the characteristics of your dataset and how vital it is for you to find every single relevant record in your set. For most, this means just setting it to a number they feel comfortable with. If you'd like to substantiate your choice for a stopping rule more and/or if it's very important for you to play it safe, you can employ the SAFE procedure Josien and Rens have developed. There is a discussion thread about stopping rules and the SAFE procedure, that you can find [here](https://github.com/asreview/asreview/discussions/557). You can quickly get started with the SAFE procedure by reading[ this pre-print article](https://osf.io/preprints/psyarxiv/c93gq) (section 2.3 stipulates the 4 steps). And when you get more familiar with the procedure, you can switch to using [this handy cheat sheet](https://ndownloader.figstatic.com/files/48529133). 

Hope this helps!
